pages/governance.html

```diff
@@ -10,193 +10,106 @@
 	</p>
+	<p>The Board of Directors currently consists of the following members:</p>
 	<ul>
-		<li>Ariel Manzur — <i>ariel@godotengine.org</i></li>
-		<li>Bastiaan Olij — <i>bastiaan@godotengine.org</i></li>
-		<li>Clay John — <i>clay@godotengine.org</i></li>
-		<li>George Marques — <i>george@godotengine.org</i></li>
-		<li>HP van Braam — <i>hp@godotengine.org</i></li>
-		<li>Ilaria Cislaghi — <i>ilaria@godotengine.org</i></li>
-		<li>Juan Linietsky — <i>juan@godotengine.org</i></li>
-		<li>Julian Murgia — <i>julian@godotengine.org</i></li>
-		<li>Rémi Verschelde — <i>remi@godotengine.org</i></li>
+		<li>Ariel Manzur — <em>ariel@godotengine.org</em></li>
+		<li>Bastiaan Olij — <em>bastiaan@godotengine.org</em></li>
+		<li>Clay John — <em>clay@godotengine.org</em></li>
+		<li>George Marques — <em>george@godotengine.org</em></li>
+		<li>HP van Braam — <em>hp@godotengine.org</em></li>
+		<li>Ilaria Cislaghi — <em>ilaria@godotengine.org</em></li>
+		<li>Juan Linietsky — <em>juan@godotengine.org</em></li>
+		<li>Julian Murgia — <em>julian@godotengine.org</em></li>
+		<li>Rémi Verschelde — <em>remi@godotengine.org</em></li>
 	</ul>
 
 	<h3 id="advisors">Advisors</h3>
 	<p>
-		The PLC has enlisted the help of its most trusted and veteran contributors
-		whom they call <em>the advisors</em>. The advisors are consulted on usage
-		of funds as well as institutional matters. It is important to note,
-		however, that while the advice of the advisory panel is weighed heavily,
-		the final authority on these matters remains with the PLC. In practice,
-		the PLC rarely makes an important decision without at least discussing it
-		with the advisors first.
+		The Board has enlisted the help of its most trusted and veteran contributors whom they call <em>the advisors</em>. The advisors are consulted on usage of funds as well as institutional matters. It is important to note, however, that while the advice of the advisory panel is weighed heavily, the final authority on these matters remains with the Board. In practice, the Board rarely makes an important decision without at least discussing it with the advisors first.
 	</p>
 	<p>
-		Like the PLC, contributors do not apply to become advisors. Instead the
-		process of joining the advisors group is organic; trusted contributors may
-		be asked to join after showing dedication, expertise, and leadership
-		within the Godot community. As a result, the advisors are a diverse group
-		of individuals with expertise in all areas of Godot development.
+		Like the Board, contributors do not apply to become advisors. Instead the process of joining the advisors group is organic; trusted contributors may be asked to join after showing dedication, expertise, and leadership within the Godot community. As a result, the advisors are a diverse group of individuals with expertise in all areas of Godot development.
```

Comment:
"As a result, the advisors are a diverse group of individuals with expertise in all areas of Godot development."

While in our case this is true, the process we outline (that is: there is no process) almost universally leads to a very non-diverse group due to selection bias. I would suggest we remove this line, or at least not imply that it is because of our process. It looks suspicious.